l10n_br_cnpj_search/models/res_partner.py

```diff
@@ -29,3 +29,19 @@ class ResPartner(models.Model):
         string="Company Currency",
         readonly=True,
     )
+
+    def action_open_cnpj_search_wizard(self):
+        if not self.name:
+            self.name = "/"
```

Comment:
aqui pode por um comentário que a gente força o sistema a por um nome temporario, para que o sistema não obrigue o usuário a digitar o nome do cliente antes de fazer a pesquisa.

Reply:
Comentário feito